kindlecomicconverter/KCC_gui.py

```diff
@@ -206,7 +206,16 @@ def clean(self):
         MW.addMessage.emit('<b>Conversion interrupted.</b>', 'error', False)
         MW.addTrayMessage.emit('Conversion interrupted.', 'Critical')
         MW.modeConvert.emit(1)
-
+        
+    def displayProgressMessage(self):
+        MW.addMessage.emit('<b>Source:</b> ' + self, 'info', False)
+        MW.addMessage.emit('Creating EPUB files... <b>Done!</b>', 'info', False)
+        GUI.progress.content = 'Creating EPUB files'
+        MW.addMessage.emit('Creating MOBI files... <b>Done!</b>', 'info', False)
+        GUI.progress.content = 'Creating MOBI files'
+        MW.addMessage.emit('Processing MOBI files... <b>Done!</b>', 'info', False)
+        GUI.progress.content = 'Processing MOBI files... <b>Done!</b>'
```

Comment:
This code block looks like it's repeating code. It also prints 'Processing MOBI' even if you are outputting to, say for example, EPUB or CBZ for a Kobo device.

Reply:
It's repeating only messages to match a process when adding files using "add files". Otherwise It would only print messages 'exclusively' for directory added using "add directory".

Reply:
The code prints out all these messages regardless of output format.
![image](https://github.com/ciromattia/kcc/assets/20757319/935e7faf-b34a-4ef8-967b-0b403d9c81bb)